web-app/src/components/Home.js

```diff
@@ -1,21 +1,24 @@
-import { QueryClient, QueryClientProvider } from 'react-query';
-import CommentaryProvider from '../contexts/commentary';
-import CommonContextProvider from '../contexts/Common';
+import { QueryClient, QueryClientProvider } from "react-query";
+import CommentaryProvider from "../contexts/commentary";
+import CommonContextProvider from "../contexts/Common";
+import DictonaryContextProvider from "../contexts/dictonary";
 import Header from "./Header";
 import VerticalTabs from "./VerticalTabs";
 
-const queryClient = new QueryClient()
+const queryClient = new QueryClient();
 
 function Home() {
-	return (
-		<QueryClientProvider client={queryClient}>
-		<CommonContextProvider>
-		<CommentaryProvider>
-				<Header />
-				<VerticalTabs />
-			</CommentaryProvider>
-		</CommonContextProvider>
-		</QueryClientProvider>
-	);
+  return (
+    <QueryClientProvider client={queryClient}>
+      <CommonContextProvider>
+        <CommentaryProvider>
+          <DictonaryContextProvider>
```

Comment:
Is DictonaryContextProvider required here cant it just be put outside the dictionary tab pane content in VerticalTabs.js
Is there any data or function required from it , if so it can be preferrably moved to CommonContextProvider